### PR TITLE
DW-4008 Added Pre Auth Lambda to Cognito User Pool

### PR DESCRIPTION
**Pull request by fernando-schmidt-dmw**

### CODE REVIEW

File: terraform/deploy/cognito/modules.tf [hunk 1]
```diff
@@ -7,6 +7,7 @@ module "cognito" {
     "jupyterhub",
   ]
 
-  root_dns_names = values(local.root_dns_name)
-  domain         = local.cognito_domain
+  root_dns_names  = values(local.root_dns_name)
+  domain          = local.cognito_domain
+  pre_auth_lambda = data.terraform_remote_state.analytical-frontend-service.outputs.pre_auth_lambda
```
Comment (IanUtley) [inline terraform/deploy/cognito/modules.tf:12]: Concourse is failing on this due to no lambdas being in front-end service currently

Comment (liamraemclauchlan) [reply terraform/deploy/cognito/modules.tf:12]: You will also need to grab the "arn" of the lambda, not the object of the lambda.

Comment (liamraemclauchlan) [reply terraform/deploy/cognito/modules.tf:12]: Fernando has another PR open to fix that - https://github.com/dwp/dataworks-analytical-frontend-service/pull/25

Comment (fernando-schmidt-dmw) [reply terraform/deploy/cognito/modules.tf]: Thanks guys.

**Pull request closed without merging**